pkg/cloud/services/compute/service.go

```diff
@@ -63,8 +64,12 @@ func NewService(scope *scope.Scope) (*Service, error) {
 	volumeClient, err := openstack.NewBlockStorageV3(scope.ProviderClient, gophercloud.EndpointOpts{
 		Region: scope.ProviderClientOpts.RegionName,
 	})
+	noVolumeService := false
 	if err != nil {
-		return nil, fmt.Errorf("failed to create volume service client: %v", err)
+		// failed to create volume service client, we should tolerate
+		scope.Logger.Info("failed to create volume service client, no volume related operations allowed")
```

Comment:
We should check what this error is, first. This would be very weird to debug if it just happened to fail with an ephemeral keystone error.

I don't have a system available without cinder. Can anybody confirm what this returns if there is no cinder?